api/src/API/Controllers/WildApricotController.cs

```diff
@@ -0,0 +1,119 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using RaceResults.Common.Models;
+using RaceResults.Data.KeyVault;
+using RestSharp;
+
+namespace RaceResults.Api.Controllers
+{
+    [ApiController]
+    [Route("wa")]
+    public class WildApricotController : ControllerBase
```

Comment:
I don't understand why we need a controller for this. The requests to the race results endpoint should validate the information after requesting user info from wild apricot and maintain the same interface with the front end.

Reply:
The backend needs to send a POST to WA to get the access token. The POST needs the secret. Hence we call this endpoint as a proxy

Reply:
That's my point. The backend is the one that needs to make these calls. We don't need to expose them as endpoints correct? In other words, the validation that happens in this endpoint can be internal right? The FE doesn't need access?

Reply:
The frontend needs to access it for two things:
1) we need to give the backend the authorization code so it can get the access token.
2) We actually can't access the WA API directly from the frontend because they don't have CORS configured. We need to proxy our requests to our server since the browser won't let the frontend talk to the WA API